src/QMCWaveFunctions/Fermion/SlaterDetBuilder.cpp

```diff
@@ -550,11 +556,18 @@ bool SlaterDetBuilder::putDeterminant(xmlNodePtr cur, int spin_group)
       adet = retval;
 #endif
     }
+#endif
+#if defined(ENABLE_CUDA)
+    else if (useGPU == "yes")
+    {
+      app_log() << "Using DiracDeterminant with DelayedUpdateCUDA engine" << std::endl;
+      adet = new DiracDeterminant<DelayedUpdateCUDA<ValueType>>(psi,firstIndex);
+    }
 #endif
     else
```

Comment:
Robustness suggestion - check for useGPU=="no", give error on other inputs. Currently "Yes" would give an unexpected result.

Reply:
I prefer not to add a check here. OhmmsAttributeSet class which passes input needs to do this work.